platform/bitcoin/api_test.go

```diff
@@ -126,16 +146,41 @@ func TestInferDirection(t *testing.T) {
 		t.Error("direction is not outgoing")
 	}
 
-	inputs2 := []string{
-		"3CgvDkzcJ7yMZe75jNBem6Bj6nkMAWwMEf",
-		"3LyzYcB54pm9EAMmzXpFfb1kzEDAFvqBgT",
-		"3Q6DYour5q5WdMhyXsyPgBeAqPCXchzCsF",
-		"3JZZM1rwst7G5izxbFL7KNvy7ZiZ47SVqG",
+	inputs2 := []blockatlas.TxOutput{
+		blockatlas.TxOutput{
+			Address: "3CgvDkzcJ7yMZe75jNBem6Bj6nkMAWwMEf",
+			Value:   "",
+		},
+		blockatlas.TxOutput{
+			Address: "3LyzYcB54pm9EAMmzXpFfb1kzEDAFvqBgT",
+			Value:   "",
+		},
+		blockatlas.TxOutput{
+			Address: "3Q6DYour5q5WdMhyXsyPgBeAqPCXchzCsF",
+			Value:   "",
+		},
+		blockatlas.TxOutput{
+			Address: "3JZZM1rwst7G5izxbFL7KNvy7ZiZ47SVqG",
+			Value:   "",
+		},
 	}
-	outputs2 := []string{
-		"139f1CrnLWvVajGzs3ZtpQhbGWxM599sho",
-		"3LyzYcB54pm9EAMmzXpFfb1kzEDAFvqBgT",
-		"bc1q9mx5tm66zs7epa4skvyuf2vfuwmtnlttj74cnl",
+	outputs2 := []blockatlas.TxOutput{
+		blockatlas.TxOutput{
```

Comment:
The same applies for all object in this array. We don't need to define the type again.

```suggestion
		{
```